### ISSUE TITLE
Why env-cmd is crashing when there is no .env file ?

### ISSUE THREAD

**Issue by gillioa** [toddbluhm/env-cmd#26]
Hi,

I was wondering why you didn't handle to make env-cmd not crashing and continue to run node when the .env file doesn't exist.

This the error I got when I try : 
```
WARNING:
      Could not find or read file at:
       /server/.env
      Trying to fallback to read:
       /server/.env

Error! Could not fallback to find or read file at /server/.env
```

I've found a way to do what I want to do but env-cmd could handle it : 
```
   "scripts": {
        "start": "env-cmd --no-override .env node . || node ."
    }
```

**Comment by toddbluhm** [comment]
@gillioa Hey sorry about this. The original default behavior of `env-cmd` was to just do nothing and run as normal, but then the fallback PR got merged in and wiped out that default behavior. 

Do you have any preference about whether to add a special flag to enable fallback or disable fallback? So something like `--fallback` or `--disable-fallback` thoughts?

**Comment by re-fort** [comment]
I have same issue.
For example, on Heroku or Circle CI, I usually set environment vars  on a setting page, not creating `.env` file.

If considering backward compatibility, it would be better to add `--disable-fallback` flag.
If not so, `--fallback` is good , IMO.

**Comment by toddbluhm** [comment]
Thanks for your support on this. I have no problem breaking backwards compatibility since I adhere to semver and have detailed change logs. I think adding a `--fallback` flag makes more sense rather than having it fallback by default. I will make that change and push a new version sometime today or tomorrow.

**Comment by re-fort** [comment]
Wow! That’s brilliant😍

**Comment by toddbluhm** [comment]
This issue has been resolved and released on npm as version `6.0.0`. If you  run into any problems with it, just open a new issue. Thanks for your comments and input everyone 😄

**Comment by re-fort** [comment]
Everything went well☺️
Thanks!!
